lib/navigation.php

```diff
@@ -418,13 +418,13 @@ function gutenberg_register_navigation_post_type() {
 		'description'           => __( 'Navigation menus.', 'gutenberg' ),
 		'public'                => false,
 		'has_archive'           => false,
-		'show_ui'               => false,
+		'show_ui'               => true,
 		'show_in_menu'          => 'themes.php',
 		'show_in_admin_bar'     => false,
 		'show_in_rest'          => true,
 		'map_meta_cap'          => true,
 		'rest_base'             => 'navigation',
-		'rest_controller_class' => 'WP_REST_Posts_Controller',
+		'rest_controller_class' => WP_REST_Posts_Controller::class,
```

Comment:
Why change this?

Reply:
IDEs like PHPStorm can understand what class is meant here if we use the `::class` keyword. This could be useful during refactoring or if we decided to use namespaces in the far future :). And it's just convenient to click on a class name to go to its definition.
I'm not aware of any modern PHP application that uses plain strings to specify class names. It just looks a little bit archaic to me.

Reply:
I think this is good but, just thinking, `::class` returns the fully qualified class name resolution, it is not "the same" as the class name as a string, and if rest_controller_class does not expect a fully qualified class name we may get into trouble.

Reply:
Yes, it does return a FQCN,  but since we don't use PHP namespaces in WordPress it equals to `'WP_REST_Posts_Controller'`. I've checked it.
And it's fail safe. Even If the `WP_REST_Posts_Controller` class is not loaded or defined, it will still return `'WP_REST_Posts_Controller'`.
I'm not insisting that we have to use `::class`, I just see no reasons not to use it.

Reply:
What happens if we do introduce Namespaces? Well ... I assume that too many things will change for this to be a big problem?

Reply:
If we do introduce namespaces, the FQCN will change to include the namespace.
E.g. instead of `'WP_REST_Posts_Controller'` it will return `'WordPress\REST\Controllers\Posts'` or something similar.
But if we introduce namespaces, we will need to know FQCN to load the correct class anyway because there could be several classes with the same name, i.e., multiple `Posts` classes in different namespaces.
This change shouldn't introduce any problems.